src/sentry/runner/commands/devserver.py

```diff
@@ -156,15 +156,27 @@ def devserver(reload, watchers, workers, experimental_spa, styleguide, prefix, e
         if eventstream.requires_post_process_forwarder():
             daemons += [
                 (
-                    "relay",
+                    "relay_events",
                     [
                         "sentry",
                         "run",
                         "post-process-forwarder",
                         "--loglevel=debug",
                         "--commit-batch-size=1",
+                        "--topic-config=events",
                     ],
-                )
+                ),
+                (
+                    "relay_transactions",
```

Comment:
This would not do anything right now, but we need all the consumer in places up and running by when we start writing to the new topic.